src/components/UserAction.tsx

```diff
@@ -79,12 +78,23 @@ interface InputProps {
 const Input = ({ data, storeAction, userAction }: InputProps) => {
   const { placeholder, type, onSubmit } = userAction
   return (
-    <input
-      className="user-action user-action-input"
-      placeholder={placeholder || ''}
-      type={type || 'text'}
-      onKeyUp={onKeyUp(data, onSubmit, storeAction)}
-    />
+    <div className="user-action">
+      <form onSubmit={handleSubmit(data, onSubmit, storeAction)}>
```

Comment:
Submitting is now only possible with clicking on submit button?

Reply:
No, you can still use the enter key. (Enter works natively to submit a form)